frontend/src/index.tsx

```diff
@@ -13,6 +13,8 @@ import { store } from '~/libs/packages/store/store.js';
 import { Auth } from '~/pages/auth/auth.js';
 import { Landing } from '~/pages/landing/landing.js';
 
+import { Profile } from './pages/profile/profile.js';
```

Comment:
Let's make pages import consistent
```suggestion
import { Profile } from '~/pages/profile/profile.js';
```